server/apps/templates/content_templates.py

```diff
@@ -143,16 +143,15 @@ def set_template_timestamps(template, now):
 
 
 def create_item_from_template(template):
-    """Create item in production using data from given template.
+    """ Create item using data from template.
 
     :param dict template
     """
-    production = superdesk.get_resource_service('archive')
     item = {key: value for key, value in template.items() if key in metadata_schema}
+    item['template'] = item.pop('_id')
     item['state'] = SCHEDULED_ITEM_STATE
```

Comment:
Can we use constants for `state`, `archive`? Also, `SCHEDULED_ITEM_STATE` is defined in `item.py.CONTENT_STATE`